stdlib/universe/group.go

```diff
@@ -392,3 +393,24 @@ func (r MergeGroupRule) Rewrite(ctx context.Context, lastGroup plan.Node) (plan.
 
 	return merged, true, nil
 }
+
+// `GroupFilterTransposeRule` orders to have 'filter |> group' from 'group |> filter'
+type GroupFilterTransposeRule struct{}
```

Comment:
Can you register this rule here? This will ensure that the rule is actually applied for the end to end tests.
https://github.com/influxdata/flux/blob/80120e492229d5fe20b4a68dbf0b13161075ccf6/stdlib/flux_test.go#L21

And also here? This will let the rule run in the Flux REPL.
https://github.com/influxdata/flux/blob/96447025b11f6a19c17cbf95c97bc40562027bd9/cmd/flux/main.go#L15